flipperdevices/flipperzero-firmware
# [FL-2438] Correct grammar in menus and application names

# What's new

- Renamed each dual component menu item with word-first capital letters, except NFC -> "Additional reading scripts"
- In NFC app renamed "Saved" section and "Save UID" action to fit naming guidelines as in other apps.

# Verification 

- Check every menu item described in Miiro board "Rename menu items" and they should be like in "Renamed and ready" section
- Check that NFC menu items "Saved cards" and "Name and save UID" changed their names to "Saved" and "Save UID"

# Checklist (For Reviewer)
